### PR TITLE
Add MeshAnything library

### PR DESCRIPTION
**Pull request by NielsRogge**
This PR ensures download stats are working for https://huggingface.co/Yiwen-ntu/MeshAnything

To do:

- [x] ensure PR is merged on repo: https://huggingface.co/Yiwen-ntu/MeshAnything/discussions/2#66894426ca6fddda11c513d4

### CODE REVIEW

File: packages/tasks/src/model-libraries-snippets.ts [hunk 1]
```diff
@@ -230,6 +230,18 @@ from inference import Mars5TTS
...
+	`git clone https://github.com/buaacyw/MeshAnything.git && cd MeshAnything
+
+conda create -n MeshAnything python==3.10.13
+conda activate MeshAnything
+pip install torch==2.1.1 torchvision==0.16.1 torchaudio==2.1.1 --index-url https://download.pytorch.org/whl/cu118
+pip install -r requirements.txt
+pip install flash-attn --no-build-isolation
+
+python app.py`,
+];
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries-snippets.ts:243]: Should we instead show something like this?

```
pip install git+https://github.com/buaacyw/MeshAnything.git

import MeshAnything
...
```

**Review by osanseviero — COMMENTED** [review]
Nice!

File: packages/tasks/src/model-libraries-snippets.ts [hunk 2]
```diff
@@ -230,6 +230,23 @@ from inference import Mars5TTS
...
+		`from MeshAnything.models.meshanything import MeshAnything
+
+# refer to https://github.com/buaacyw/MeshAnything/blob/main/main.py#L91 on how to define args
+# and https://github.com/buaacyw/MeshAnything/blob/main/app.py regarding usage
+model = MeshAnything(args)`,
+	];
+
+	return [shellCommands.join("\n"), pythonCode.join("\n")];
+};
+];
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries-snippets.ts:248]:
```suggestion
export const mars5_tts = (model: ModelData): string[] => [
	`# Install from https://github.com/buaacyw/MeshAnything.git

from MeshAnything.models.meshanything import MeshAnything

# refer to https://github.com/buaacyw/MeshAnything/blob/main/main.py#L91 on how to define args
# and https://github.com/buaacyw/MeshAnything/blob/main/app.py regarding usage
model = MeshAnything(args)`,
];
```

I would combine install comment + python script in one snippet (same as [mars5-TTS](https://github.com/huggingface/huggingface.js/blob/62d6aaa6e01fa558c70745e516c0c4e854daed41/packages/tasks/src/model-libraries-snippets.ts#L226C1-L231C3))

Comment (NielsRogge) [reply packages/tasks/src/model-libraries-snippets.ts:248]: Haha ok Pedro had a comment to split them here: https://github.com/huggingface/huggingface.js/pull/785#discussion_r1667652837, which is why I adopted the same here.

Comment (Wauplin) [reply packages/tasks/src/model-libraries-snippets.ts:248]: Ahah yes, understandable. But I think we should not show the shell commands at all to simplify the snippet and invite users to correctly install the repo instead.

**Comment by NielsRogge** [comment]
Feel free to approve as https://huggingface.co/Yiwen-ntu/MeshAnything/discussions/2#66894426ca6fddda11c513d4 has been merged

**Review by Wauplin — APPROVED** [review]
Thanks @NielsRogge!

**Review by osanseviero — APPROVED** [review]
LGTM!

**Pull request merged**